qiskit_ibm_runtime/fake_provider/fake_backend.py

```diff
@@ -296,8 +294,13 @@ def target(self) -> Target:
         return self._target
 
     @property
-    def max_circuits(self) -> None:
-        return None
+    def max_circuits(self) -> int:
+        """The maximum number of circuits
+
+        The maximum number of circuits that can be
+        run in a single job. If there is no limit this will return None.
```

Comment:
```suggestion
        The maximum number of circuits that the backend supports in a single execution. Note that the actual number of circuits the service allows may be different. 
```

Because we don't actually use that number anymore 😛. In fact we should just deprecate this attribute.